avm/res/sql/server/main.bicep

```diff
@@ -485,6 +488,41 @@ module server_audit_settings 'audit-settings/main.bicep' = if (!empty(auditSetti
   }
 }
 
+module server_secrets_export 'modules/key-vault-export.bicep' = if (!empty(secretsExportConfiguration)) {
```

Comment:
This change should be put on hold until https://github.com/Azure/bicep-registry-modules/issues/1934 is resolved and the spec defined. @jtracey93, added it for the upcoming discussion call. Shouldn't take long to go through the suggestions, agree to one and define the spec. There are also a few other PRs that were created similar to this one in the past:
- #1920 
- #1932 
- #1950

We have to get this spec out there asap.